src/plugins/htmlVideoPlayer/style.scss

```
@@ -74,6 +74,15 @@ video[controls]::-webkit-media-controls {
     display: inline-block;
 }
 
+.videoSecondarySubtitlesInner {
+    max-width: 70%;
+    background-color: rgba(0, 0, 0, 0.8);
+    margin: auto;
+    display: block;
+    min-height: 0 !important;
+    margin-bottom: 0.5em !important;
```

see also the post about `setSubtitleAppearance`.
```suggestion
    margin-bottom: 0.5em !important;
    margin-top: 0.5em !important;
```